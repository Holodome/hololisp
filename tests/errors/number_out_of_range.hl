

    123312389219389213981289312893891238918923
