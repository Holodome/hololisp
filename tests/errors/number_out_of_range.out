cli:3:5: error: Number literal is out of range
    123312389219389213981289312893891238918923
    ^
1 error generated.
